src/app/route/CheckoutPage/CheckoutPage.component.js

```diff
@@ -0,0 +1,625 @@
+                },
+                klarna: {
+                    title: "Klarna smooth payments"
+                }
+            }
+        };
+    }
+
+    componentDidMount() {
+    }
+
+    componentDidUpdate(prevProps) {
+    }
+
+    renderShippingMethod(identifier, value) {
+        const { shippingMethod } = this.state;
+
+        return (
+            <tr key={ identifier } onClick={ () => this.setState({ shippingMethod: identifier }) }>
+                <td>
+                    <Field
+                        id={ identifier }
+                        type="radio"
+                        block="shippingMethodTable"
+                        elem={ identifier }
+                        value={ identifier }
+                        checked={ shippingMethod }
+                        onChange={ identifier => this.setState({ shippingMethod: identifier }) }
+
+                    />
+                </td>
+                <td>{ value.price }</td>
+                <td>{ value.title }</td>
+                <td>{ value.carrier_title }</td>
+            </tr>
+        );
+    }
+
+    renderPaymentMethod(index, data) {
+        const { paymentMethod } = this.state;
+        return (
+            <tr key={ index } onClick={ () => this.setState({ paymentMethod: index }) }>
+                <td>
+                    <Field
+                        id={ index }
+                        type="radio"
+                        block="paymentMethodTable"
+                        elem={ index }
+                        value={ index }
+                        checked={ paymentMethod }
+                        onChange={ index => this.setState({ paymentMethod: index }) }
+                    />
+                </td>
+                <td>{ data.title }</td>
+            </tr>
+        );
+    }
+
+    placeOrder() {
+        //todo
+    }
+
+
+    render() {
+        const {
+            checkoutStep,
+            email,
+            shippingAddress,
+            shippingMethod,
+            shippingMethods,
+            paymentMethods,
+            billingIsSame,
+            billingAddress,
+        } = this.state;
+
+        const {
+            products,
+            totals
+        } = this.props;
+
+        return (
+            <main block="CheckoutPage">
+                <ContentWrapper
+                    wrapperMix={ { block: 'CheckoutPage', elem: 'Wrapper' } }
+                    label="Checkout page"
+                >
+                    <div
+                     block="CheckoutPage"
+                     elem="CheckoutSteps"
+                    >
+                        <h1 block="CheckoutPage" elem="Heading">
+                            Checkout
+                        </h1>
+
+                        { checkoutStep === CHECKOUT_STEP_SHIPPING ? (
+                            <div block="checkoutStep" elem="shippingStep">
+                                <fieldset block="CheckoutStep" elem="legend">
```

Comment:
We might need this billing / shipping address logic one again in the MyAccount. Please make these fields a component and provide enough abstraction for easy manipulation later on.